src/java-runtime/index.ts

```diff
@@ -175,14 +181,18 @@ export async function findJavaRuntimeEntries(): Promise<JavaRuntimeEntry[]> {
   return Promise.all(_.map(entries, async entry => {
     if (!await checkJdkInstallation(entry.path)) {
       entry.isValid = false;
-      entry.hint = "This path does not point to a JDK.";
+      entry.hint = "This path is not pointing to a JDK.";
+      const pathObject = path.parse(entry.path||"");
+      if (pathObject.name && pathObject.name === "bin") {
```

Comment:
nit: `pathObject.name === "bin"` is enough. Leave the choice to you

Reply:
Addressed. Please check the new commit.